mungegithub/submit-queue/deployment/contrib/configmap.yaml

```diff
@@ -32,6 +32,9 @@ data:
   submit-queue.chart-url: ""
   submit-queue.history-url: ""
   submit-queue.batch-url: ""
+  # It doesn't seem that this is correct even though the contrib submit queue
+  # today points to this url.
+  submit-queue.context-url: https://submit-queue.k8s.io
```

Comment:
This should be https://contrib.submit-queue.k8s.io  It looks like the links the contrib submit queue currently uses for its status context don't work.